### PR TITLE
Fe 11 - Create Home Page and Blueprints Page as demos with mock data

### PR DESCRIPTION
**Pull request by IvoKarapeev**
Created Components:
- Button 
- Carousel
- Header
- ImageContent
- Input
- Sidebar
- Text
- Toggle
- BlueprintItems

Created Demo Pages:
- Blueprints
- HomePage

Helpers Added: 
- mock-data.ts - used for mock data for the new Demo Pages

Store:
- Added theme store ( themeSlice.ts ) for the theme ( dark || light ) of the Application
- Created store.ts to have both ( authSlice and themeSlice ) in one global store

Styles:
- Added scss in the project
- Added utility classes ( used in the demo pages ) - utility classes are created for us to be able to add some repeatable styles with this classes
- Added scss variables

NPM Packages:
- sass
- react-router-dom
- classnames
- framer-motion

### CODE REVIEW

File: we-smooth-fe/src/helpers/mock-data.ts
Comment (bobi591) [inline we-smooth-fe/src/helpers/mock-data.ts:1]: very helpful for backend

File: we-smooth-fe/src/components/auth-context/login.tsx [hunk 1]
```diff
@@ -20,7 +20,7 @@ const Login: FunctionComponent = () => {
     }, [])
 
     return (
-        <>
+        <div style={{ position: 'absolute', top: '65px' }}>
```
Comment (bobi591) [inline we-smooth-fe/src/components/auth-context/login.tsx:23]: at some point we should give this a className and have stylise it scss file

**Comment by bobi591** [comment]
You did a lot of things, and most of them are very complicated. Great job!

There are no changes in the configuration of the project which makes me think you didn't had issues with Lint which I was not expecting honestly.

Only we should get rid of these SVGs at some point, they are bloating the code.

Let me test it locally in the upcoming days and I will approve or drop a comment.

File: we-smooth-fe/src/components/auth-context/login.tsx (hunk 1 above)
Comment (IvoKarapeev) [reply we-smooth-fe/src/components/auth-context/login.tsx:23]: Yes, of course. Given that the login page component is currently being designed, I've decided to hold off on adding the SCSS styling for now. Once its design is ready, we can proceed with implementing the component and its styles.

File: we-smooth-fe/src/helpers/mock-data.ts
Comment (IvoKarapeev) [reply we-smooth-fe/src/helpers/mock-data.ts:1]: Glad to hear that!

**Pull request merged**